Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.cs

```diff
@@ -182,6 +182,37 @@ private void EnsureThereIsNoOpenReactivationRequest()
             throw new DomainException(DomainErrors.CannotRequestReactivationWhenThereIsAnOpenReactivationRequest());
     }
 
+    public void AcceptReactivation(IdentityAddress activeIdentity, DeviceId activeDevice)
+    {
+        EnsureOpenRevivalRequestExists();
+        EnsureRelationshipReactivationRequestIsAddressedToSelf(activeIdentity);
```

Comment:
Check out Nikola V's `EnsureRevocableReactivationRequestExistsFor` method and do it similarly, instead of these two calls.

Reply:
done